README.md

```diff
@@ -1,5 +1,7 @@
 ![phoenix logo](https://raw.githubusercontent.com/phoenixframework/phoenix/master/priv/static/phoenix.png)
-> Productive. Reliable. Fast. Most web frameworks make you choose between speed and a productive environment. Phoenix gives you both.
+> ### Productive. Reliable. Fast.
+> A productive web framework that
+> does not compromise speed and maintainability
```

Comment:
@chrismccord this didn't line break like your comment, what fancy trick did you use?